app/controllers/authenticate_controller.rb

```diff
@@ -207,9 +207,9 @@ def status_failure_response(error)
     status_code = case error
                   when Errors::Authentication::Security::RoleNotAuthorizedOnResource
                     :forbidden
-                  when Errors::Authentication::StatusNotImplemented
+                  when Errors::Authentication::StatusNotSupported
```

Comment:
you have tests who use `StatusNotImplemented` (in validate_status_spec.rb) please change those as well